test/test-worktree.sh

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env bash
+
+. "test/testlib.sh"
+
+# git 2.5 or higher only
+compare_version $(git version | cut -d" " -f3) "2.5.0"
+if [[ $? != $VERSION_LOWER ]]; then
+  begin_test "git worktree"
```

Comment:
Any thoughts on just exiting early before `begin_test()` to remove the extra level of nesting? Maybe even spit out an error message like "skipped worktree test because git {your_version} < 2.5.0".